scripts/variantstore/docs/aou/AOU_DELIVERABLES.md

```diff
@@ -99,9 +99,21 @@ The Callset Stats and S&P files can be simply `gsutil cp`ed to the AoU delivery
 ## Running the VAT pipeline
 To create a BigQuery table of variant annotations, you may follow the instructions here:
 [process to create variant annotations table](../../variant_annotations_table/README.md)
-
 The pipeline takes in the VDS and outputs a variant annotations table in BigQuery.
 
+Once the VAT table and a tsv is exported, the AoU research workbench team should be notified of its creation and permission should be granted so that several members of the team have view permission.
+
+- Grant `BigQuery Data Viewer` permission to specific people's PMI-OPS accounts. As of the Echo release, these accounts were:
+        <ul>
+            <li>kevin.levy@pmi-ops.org</li>
+            <li>srushti.gangireddy@pmi-ops.org</li>
+            <li>brian.freeman@pmi-ops.org </li>
+            <li>tarek.ahmed@pmi-ops.org</li>
+            <li>jennifer.zhang@pmi-ops.org</li>
+        </ul>
+- Copy the bgzipped, tarred export of the VAT into the pre-delivery bucket. For the Echo release, it was copied to `gs://prod-drc-broad/v8/wgs/vat/aou_srwgs_short_variants_v8r1.vat.bgz.tsv.gz`
+- Finally, an email should be sent out notifying the AoU research workbench team (and others) of the readiness of the VAT. For echo, the email had the subject line: `Variant Annotations Table delivered for v8 srWGS joint callset` (it may just be easiest to find that email and modify it accordingly for the release you are working on.)
```

Comment:
```suggestion
- Finally, an email should be sent out notifying the AoU research workbench team (and others) of the readiness of the VAT. For echo, the email had the subject line: `Variant Annotations Table delivered for v8 srWGS joint callset` (it may just be easiest to find that email and modify it accordingly for the release you are working on).
```